app/src/main/java/org/wikipedia/analytics/eventplatform/EventPlatformClient.java

```diff
@@ -438,9 +433,11 @@ private static void refreshStreamConfigs() {
     /*
      * The constructor is private, so instantiation from other classes is impossible.
      */
-    private EventPlatformClient() {
+    public static void setUpStreamConfigs() {
```

Comment:
Would it be possible to just name it as `setup()` since it is the way to setup the MEP client?

Reply:
@cooltey stream configs is just a part of the client so I feel this is more readable.